apps/metadata-editor/src/app/my-org-users/my-org-users.component.spec.ts

```diff
@@ -59,43 +47,32 @@ describe('MyOrgUsersComponent', () => {
       }),
     }
 
-    const authServiceMock = {
-      user$: new BehaviorSubject(user),
-      allUsers$: new BehaviorSubject(allUsers),
-    }
-
-    const organisationsServiceMock = {
-      organisations$: of(ORGANISATIONS_FIXTURE),
-    }
-
-    const searchFacadeMock = {
-      resetSearch: jest.fn(),
-    }
-
     myOrgService = myOrgServiceMock as any
-    authService = authServiceMock as any
-    searchFacade = searchFacadeMock as any
-
-    component = new MyOrgUsersComponent(myOrgService, searchFacade)
+    component = new MyOrgUsersComponent(myOrgService)
   })
 
   it('should create', () => {
     expect(component).toBeTruthy()
   })
 
   describe('Get organization users info', () => {
+    let orgData
+
+    beforeEach(() => {
+      orgData = null
+      component.orgData$.subscribe((data) => (orgData = data))
+    })
+
     it('should get the org name', () => {
-      expect(component.orgData.orgName).toEqual('wizard-org')
+      expect(orgData.orgName).toEqual('wizard-org')
```

Comment:
This feels like you are testing the data type, and not the rendering. Why not select the html element bearing the dynamic content and test that this is correct?

Reply:
the users will be fed to another component; but really even this test is a bit superfluous since there's barely any logic in this component.